libbeat/docs/shared-template-load.asciidoc

```diff
@@ -101,27 +126,78 @@ and run:
 
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
-PS > {beatname_lc} setup --template
+PS > {beatname_lc} setup --template -E output.logstash.enabled=false -E 'output.elasticsearch.hosts=["localhost:9200"]'
 ----------------------------------------------------------------------
 
 
-[NOTE]
-=====
+[[force-kibana-new]]
+===== Force Kibana to look at newest documents
+
 If you've already used {beatname_uc} to index data into Elasticsearch,
 the index may contain old documents. After you load the index template,
 you can delete the old documents from +{beatname_lc}-*+ to force Kibana to look
 at the newest documents. Use this command:
 
+*deb, rpm, and mac:*
+
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
 curl -XDELETE 'http://localhost:9200/{beatname_lc}-*'
 ----------------------------------------------------------------------
 
-This command will delete all indexes that match the pattern +{beatname_lc}-*+.
-Before running this command, make sure you want to delete all indexes that
-match the pattern.
+*win:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+PS > Invoke-RestMethod -Method Delete "http://localhost:9200/{beatname_lc}-*"
+----------------------------------------------------------------------
+
+
+This command deletes all indexes that match the pattern +{beatname_lc}-*+.
+Before running this command, make sure you want to delete all indexes that match
+the pattern.
+
+[[load-template-manually-alternate]]
+==== Load the template manually (alternate method)
+
+If the host running {beatname_uc} does not have direct connectivity to
+Elasticsearch, you can export the index template to a file and then install the
```

Comment:
Maybe it won't be obvious to users that they need to move or copy it to a different machine with ES connectivity. Should we add this into this sentence (e.g. export, transfer, and then install)?